#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_vboxnetflt.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not build vboxnetflt module.

@DPATCH@

--- virtualbox-ose-2.1.0-dfsg/src/VBox/HostDrivers/VBoxNetFlt/Makefile.kmk.orig	2008-12-23 15:40:38.000000000 +0100
+++ virtualbox-ose-2.1.0-dfsg/src/VBox/HostDrivers/VBoxNetFlt/Makefile.kmk	2008-12-23 15:39:34.000000000 +0100
@@ -221,7 +221,7 @@
 # vboxnetflt(.ko/.o/) - The lower case driver.
 # Note! On Solaris the name has to be <= 8 chars long.
 #
-SYSMODS += vboxnetflt
+#SYSMODS += vboxnetflt
 vboxnetflt_TEMPLATE      = VBOXR0DRV
 vboxnetflt_NAME.solaris  = vboxflt
 vboxnetflt_DEFS.linux    = KBUILD_MODNAME=KBUILD_STR\(vboxnetflt\) KBUILD_BASENAME=KBUILD_STR\(vboxnetflt\) MODULE
